src/commands/base-command.js

```diff
@@ -300,13 +300,14 @@ class BaseCommand extends Command {
     const duration = getDuration(startTime)
     const status = error_ === undefined ? 'success' : 'error'
 
-    debug(`${this.name()}:onEnd`)(`Status: ${status}`)
-    debug(`${this.name()}:onEnd`)(`Duration: ${duration}ms`)
+    const command = Array.isArray(this.args) ? this.args[0] : this.name()
```

Comment:
This seems rather defensive, but I rather not have the CLI error if something changes with `commander`